pvr.hts/resources/settings.xml

```diff
@@ -2,6 +2,7 @@
 <settings>
   <category label="30000">
     <setting id="host" type="text" label="30001" default="127.0.0.1" />
+    <setting id="https" type="bool" label="30018" default="false" />
```

Comment:
Please use id 30011 and add it here https://github.com/kodi-pvr/pvr.hts/blob/master/pvr.hts/resources/language/resource.language.en_gb/strings.po#L65

Reply:
Done.